assembly/pallets/timestamp/assembly/timestamp.ts

```diff
@@ -0,0 +1,155 @@
+import { Storage, Log, IInherent } from 'subsembly-core';
+import { InherentData, Inherent, ResponseCodes } from 'subsembly-core';
+import { Utils } from "subsembly-core";
+import { Bool, ByteArray, BytesReader } from 'as-scale-codec';
+import { Moment } from '../../../runtime/runtime';
+
+export class Timestamp{
```

Comment:
I think that we should have inline documentation for this class.

Reply:
We can think about what variable should be user-defined and what should be "hardcoded"